File: work-spring/BoardOAuth/src/main/java/com/grepp/boardjwt/model/repository/TodoRepository.java
package com.grepp.boardjwt.model.repository;


import com.grepp.boardjwt.model.entity.TodoEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface TodoRepository extends JpaRepository<TodoEntity, String> {
  List<TodoEntity> findByUsername(String username);

//  @Query("SELECT t FROM todo t WHERE t.username = ?1")
//  TodoEntity findByUserIdQuery(String userId);
  TodoEntity findById(int id);
}

File: work-kotlin/day1021_todo/src/main/kotlin/com/grepp/todo/model/repository/TodoRepository.kt
package com.grepp.todo.model.repository

import com.grepp.todo.model.entity.TodoEntity
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface TodoRepository :CrudRepository<TodoEntity, Long> {
    // todo 추가 : insert 기본
    // todo 조회
    fun findByUsername(username:String) : List<TodoEntity>

}